src/accessibility/WCAG-2.2/index.md

```diff
@@ -0,0 +1,393 @@
+
+### What you need to do
+
+1. Revisit the [Government Digital Service (GDS) guidance](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps) on accessibility is and why your service needs to invest in it
+2. Read [What’s new in WCAG 2.2](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/) to understand the new criteria your service will need to comply with
+3. Ask your wider organisation for support to prepare for work needed to meet WCAG 2.2
+4. Review the lists on this page of affected components and patterns in the Design System
+5. Assess your service to see how many affected components and patterns are in use
+6. Plan and implement the changes needed to your service
+
+Make sure there's expertise within your organisation by advocating for people to receive training in accessibility. To provide some help with this, the Design System team is [organising community events](/community/) to help service teams share information with each other.
+
+### Components and patterns affected in the Design System
+
+We've made changes to these components and patterns to comply with WCAG 2.2 level AA. You must check if your service needs amending to align with these changes.
+
+[Images](/styles/images/) also includes guidance to help ensure icons and images in your service meet Target Size (minimum).
+
+{{ govukTable({
+  caption: "Components",
+  captionClasses: "govuk-table__caption--m",
+  head: [
+    {
+      text: "Component",
+      classes: "govuk-!-width-one-half"
+    },
+    {
+      text: "Relevant WCAG 2.2 criteria",
+      classes: "govuk-!-width-one-half"
+    }
+  ],
+  rows: [
+    [
+      {
+        html: '<a href="/components/back-link/">Back link</a>'
+      },
+      {
+        html: 'Redundant entry<br>Target size (minimum)'
+      }
+    ],
+    [
+      {
+        html: '<a href="/components/breadcrumbs/">Breadcrumbs</a>'
+      },
+      {
+        html: 'Target size (minimum)'
+      }
+    ],
+    [
+      {
+        html: '<a href="/components/button/">Button</a>'
+      },
+      {
+        html: 'Target size (minimum)'
+      }
+    ],
+    [
+      {
+        html: '<a href="/components/cookie-banner/">Cookie banner</a>'
+      },
+      {
+        html: 'Focus not obscured (minimum)<br>Target size (minimum)'
+      }
+    ],
+    [
+      {
+        html: '<a href="/components/error-message/">Error message</a>'
+      },
+      {
+        html: 'Redundant entry'
+      }
+    ],
+    [
+      {
+        html: '<a href="/components/file-upload/">File upload</a>'
+      },
+      {
+        html: 'Dragging movements<br>Redundant entry'
+      }
+    ],
+    [
+      {
+        html: '<a href="/components/footer/">Footer</a>'
+      },
+      {
+        html: 'Consistent help'
+      }
+    ],
+    [
+      {
+        html: '<a href="/components/header/">Header</a>'
+      },
+      {
+        html: 'Focus not obscured (Minimum)<br>Focus appearance'
```

Comment:
What's 'Consistent help' on this page is wrongly listed as 'Focus appearance' on the overview page.

```suggestion
        html: 'Focus not obscured (Minimum)<br>Consistent help'
```